ansible/roles/docker/docker-image/tasks/main.yml

```diff
@@ -46,12 +46,12 @@
       - docker_password is defined
       - not aws_login
 
-  - name: Reading path to Python repository with Docker package
-    command: "{{ ansible_python_interpreter }} -c \"import docker as _; print(_.__file__.replace('/docker/__init__.py', ''))\""
+  - name: Reading path to Requests repository to find packages env
+    command: "{{ ansible_python_interpreter }} -c \"import requests as _; print(_.__file__.replace('/requests/__init__.py', ''))\""
```

Comment:
sketch af :grimacing:

Reply:
actually i don't know that, but it certainly looks sketchy... what's this doing?

Reply:
finding the location of the "requests" package and then going up dir to find the python packages path. Not completely certain that python will always be able to find the docker module like this (as docker is installed as an ansible collection instead of via pip), so it seemed safer to find the requests module instead